### PR TITLE
add tree-sitter CLI

### PR DESCRIPTION
**Pull request by mrjones2014**
I think I did this correctly, happy to adjust as necessary.

### CODE REVIEW

**Comment by williambotman** [comment]
@mason-org/triage `notify(Reason::NewPullRequest)`

<sup>This comment ensures a notification is delivered to maintainers.</sup>

<sup>` 🤖 This is an automated comment. `  [` 📖 Source code `](https://github.com/williamboman/botman)</sup>

**Comment by mrjones2014** [comment]
I'll defer to your judgement and change it 🙂 

Will update shortly.

**Comment by mrjones2014** [comment]
Actually, I noticed that tree-sitter CLI does not offer a macOS ARM pre-built binary, only macOS x64. I'll push the commit so you can take a look at it, but based on this information, it may actually be better to use `cargo`, since compiling locally will produce a native binary for macOS ARM.

**Comment by williamboman** [comment]
@williambotman /apply

```diff
diff --git a/packages/treesitter-cli/package.yml b/packages/treesitter-cli/package.yml
index a8b76ee..5e7e688 100644
--- a/packages/treesitter-cli/package.yml
+++ b/packages/treesitter-cli/package.yml
@@ -1,25 +1,35 @@
 ---
 name: treesitter-cli
-description: The Tree-sitter CLI allows you to develop, test, and use Tree-sitter grammars from the command line. It works on MacOS, Linux, and Windows.
-homepage: https://github.com/tree-sitter/tree-sitter/blob/master/cli/README.md
+description: |
+  The Tree-sitter CLI allows you to develop, test, and use Tree-sitter grammars from the command line. It works on
+  MacOS, Linux, and Windows.
+homepage: https://tree-sitter.github.io/tree-sitter/
 licenses:
-    - MIT
+  - MIT
 categories:
-    - Compiler
+  - Compiler
 
 source:
-    id: pkg:github/tree-sitter/tree-sitter@0.20.8
-    asset:
-    - target: darwin_x64
-      file: tree-sitter-macos-x64.gz
-    - target: darwin_arm64
+  id: pkg:github/tree-sitter/tree-sitter@0.20.8
+  asset:
+    - target: [darwin_x64, darwin_arm64]
       file: tree-sitter-macos-x64.gz
+      bin: tree-sitter-macos-x64
     - target: linux_x64
       file: tree-sitter-linux-x64.gz
+      bin: tree-sitter-linux-x64
+    - target: linux_x86
+      file: tree-sitter-linux-x86.gz
+      bin: tree-sitter-linux-x86
     - target: linux_arm64
       file: tree-sitter-linux-aarch64.gz
+      bin: tree-sitter-linux-aarch64
     - target: win_x64
-      file: tree-sitter-windows-x64.gz
+      file: tree-sitter-windows-x64.gz:tree-sitter-windows-x64.exe.gz
+      bin: tree-sitter-windows-x64.exe
+    - target: win_x86
+      file: tree-sitter-windows-x86.gz:tree-sitter-windows-x86.exe.gz
+      bin: tree-sitter-windows-x86.exe
 
 bin:
-    tree-sitter: cargo:tree-sitter-cli
+  tree-sitter: "{{source.asset.bin}}"
```

**Review by williamboman — APPROVED** [review]
> Actually, I noticed that tree-sitter CLI does not offer a macOS ARM pre-built binary, only macOS x64. I'll push the commit so you can take a look at it, but based on this information, it may actually be better to use cargo, since compiling locally will produce a native binary for macOS ARM.

That's fine, I think it's a fair trade off.

I changed indentation, added some more targets (x86) and fixed the bin link. LGTM!

**Comment by williamboman** [comment]
@williambotman /apply

```diff
diff --git a/packages/treesitter-cli/package.yml b/packages/tree-sitter-cli/package.yml
similarity index 90%
rename from packages/treesitter-cli/package.yml
rename to packages/tree-sitter-cli/package.yml
index 5e7e688..e12f24e 100644
--- a/packages/treesitter-cli/package.yml
+++ b/packages/tree-sitter-cli/package.yml
@@ -1,9 +1,9 @@
 ---
-name: treesitter-cli
+name: tree-sitter-cli
 description: |
   The Tree-sitter CLI allows you to develop, test, and use Tree-sitter grammars from the command line. It works on
   MacOS, Linux, and Windows.
-homepage: https://tree-sitter.github.io/tree-sitter/
+homepage: https://github.com/tree-sitter/tree-sitter/blob/master/cli/README.md
 licenses:
   - MIT
 categories:

```

**Comment by williamboman** [comment]
Needs to merge with upstream to fix CI failure. Waiting for @williambotman to redeploy to latest version and after that he'll take care of it!

**Comment by williamboman** [comment]
@williambotman /merge-base

**Comment by williamboman** [comment]
Ah no so the CI was correct, the file extension should be `.yaml`, fixing

**Comment by williamboman** [comment]
@williambotman /apply

```diff
diff --git a/packages/tree-sitter-cli/package.yml b/packages/tree-sitter-cli/package.yaml
similarity index 100%
rename from packages/tree-sitter-cli/package.yml
rename to packages/tree-sitter-cli/package.yaml
```

File: packages/tree-sitter-cli/package.yaml
Comment (williamboman) [inline packages/tree-sitter-cli/package.yaml:8]:
```suggestion
  - MIT
languages: []  
```

File: packages/tree-sitter-cli/package.yaml
Comment (williamboman) [inline packages/tree-sitter-cli/package.yaml:14]:
```suggestion
  id: pkg:github/tree-sitter/tree-sitter@v0.20.8
```

**Pull request merged**
